closure/testing/test/BUILD

```diff
@@ -111,6 +111,40 @@ closure_js_test(
     ],
 )
 
+# TEST: closure_js_library targets from exports should be provided when used in
+# a data attr.
+
+closure_js_library(
+    name = "external",
+    srcs = ["external.js"],
+)
+
+closure_js_library(
+    name = "all_external",
+    exports = [":external"],
+)
+
+closure_js_test(
+    name = "external_code_test",
+    srcs = ["external_code_test.js"],
+    html = "external_code_test.html",
+    entry_points = [
+        "rules_closure.ExternalCodeTest",
+    ],
+    data = [
+        # including external.js through a rollup target via exports should work.
+        ":all_external",
+    ],
+    # NOTE: I'm not sure why this is necessary.
+    suppress = [
+        "reportUnknownTypes",
```

Comment:
Is this still needed? What is unknown here?

Reply:
It's not still needed. Removing